### DISCUSSION TITLE
Registration required to view logs for github hosted app?

### DISCUSSION THREAD

**Discussion by chezsmithy** [renovatebot/renovate#25549] [Request Help] [answered]
Noted that there has been a change when viewing logs from the github hosted app. When requesting a log mend.io site is now requesting the user to supply an email, agree to terms of service / privacy policy. Is this expected?

Also noting the terms suggest the person registering is accepting terms on behalf of the entire organization which raised concerns.

Can these new terms be accepted centrally by an admin and have this step skipped for log viewers?

**Comment by rarkins** [comment, marked as answer]
The UI changed but the registration step was a part of the old site too. Each user needs to agree to ToS including privacy policy individually and there's no way around this.

**Reply by chezsmithy** [reply]
@rarkins got it. When you visited logs directly from a PR on GitHub created by Renovate it used to open straight to the logs vs requiring the registration step. I believe that has changed now.

**Reply by rarkins** [reply]
I can't guarantee that there wasn't ever a _bug_, but the intention has always been that users needed to agree to the ToS _once_ and then never again. GitHub's ToS/Privacy terms do not extend to third parties, so it's essentially from a legal perspective that users consent to the terms and that we have an email address we can reach people in case we need to change terms, notify of important information, etc. I don't think there's any web app or service in the world which lets you create an account or use personalization of any sort without agreeing to the site's conditions.

**Reply by chezsmithy** [reply]
Ok. Completely reasonable. Thanks for the explanation!
